olm-catalog/serverless-operator/project.yaml

```diff
@@ -9,7 +9,7 @@ olm:
   channels:
     default: '4.6'
     list:
-      - '4.6'
+      - '4.6,4.7'
```

Comment:
This must actually be an enumerated list. This breaks the code figuring out which channel to use for the subscription, see https://github.com/openshift-knative/serverless-operator/blob/28c6716b70c438a15ae747fd6c2e3ce7ed472350/hack/lib/vars.bash#L57.